task/github.py

```diff
@@ -121,17 +122,24 @@ def __init__(self, base=None, cacher=None, repo=None):
         self.token = None
         self.debug = False
         try:
-            with open(xdg_config_home('github-token'), "r") as f:
+            with open(xdg_config_home('cockpit-dev', 'github-token'), "r") as f:
```

Comment:
I have to do some bikeshedding here: There is actually nothing cockpit specific about this token. It represents *you* in GitHub, not any particular project. It equally applies when you are hacking on subscription-manager, lorax, starter-kit and all other projects that use our bots project; and you could use it in your own scripts just as well. So unlike image-stores, I don't consider this particularly bad namespace trampling honestly.

If you go through with that, please adjust README and .github/workflows/daily.yml alongside. After that we should then also gradually adjust the workflows and scripts that use it in cockpit, starter-kit, cockpit-podman, etc.

Reply:
This is not 100% true.

You can create any number of tokens in github.  Back in the day, before I even understood any of this stuff, I created mine with exactly the permissions that were required for cockpit, and named the token "cockpit bots".  At least in my case, it's very cockpit-specific.

My `gh` has a token with more capabilities.